app/models/npq_lead_provider.rb

```diff
@@ -3,6 +3,6 @@
 class NPQLeadProvider < ApplicationRecord
   belongs_to :cpd_lead_provider, optional: true
 
-  has_many :npq_profiles, class_name: "NPQApplication"
-  has_many :npq_participants, through: :npq_profiles, source: :user
+  has_many :npq_applications, class_name: "NPQApplication"
+  has_many :npq_participants, through: :npq_applications, source: :user
```

Comment:
Is this right? Shouldn't we separate who are actually npq_participants and what is npq_applications?

Reply:
I think you are right - this should be something like 
```
  has_many :npq_applications, class_name: "NPQApplication"
  has_many :npq_participant_profiles, class_name: "ParticipantProfile::NPQ", through: :npq_applications, source: :profile
  has_many :npq_participants, through: :npq_participant_profiles, source: :user
```

Reply:
Logic on those associations feels a little bit backwards, but I get that the profile is created from the application and not the other way around.